delivery/client.py

```diff
@@ -43,3 +52,31 @@ def component_descriptor(
             res.json(),
             validation_mode=validation_mode,
         )
+
+    def post_compliance_scan(
```

Comment:
maybe slightly different than stated before: it is okay-ish to have this generic, tool-agnostic "lowlevel" posting-method.

however, as stated before, this method should accept component + artifact (rather than flattened attrs from those). of course, at this point, those attrs will have to be flattened into a json (but this should happen inside this method).

also, either add tool-specific (convenience) methods that will then delegate to this method, or implement conversion gear that will translate tool-specific scan-results into arg-vectors for this method